packages/core/src/stylable-processor.ts

```diff
@@ -346,48 +352,35 @@ export class StylableProcessor {
                         this.meta.imports.push(stImport);
                         this.addImportSymbols(stImport);
                     }
-
                     break;
-                case 'property':
-                    this.checkRedeclareSymbol(atRule.params, atRule);
+                }
+                case 'property': {
                     this.addCSSVarDefinition(atRule);
+
+                    const { valid, message, remove } = validateAtProperty(atRule);
+
+                    if (!valid && message) {
+                        this.diagnostics.warn(atRule, message, { word: atRule.params });
+                    }
+
+                    if (remove) {
+                        toRemove.push(atRule);
```

Comment:
move to the end of the transformer flow, we don't want to add more changes in the processor